rep-I0001.rst

```diff
@@ -109,9 +108,11 @@ The dynamic joint state is meant to mimic both the ROS JointState and FollowJoin
         accelerations[] 
         effort[]
         position_desired[]
-        position_errors[]
+        position_error[]
         velocity_desired[]
-        velocity_errors[]
+        velocity_error[]
+        accel_desired[]
+        accel_error[]
```

Comment:
For brevity in typing.  {shrug}.  In reality, these field names don't officially appear anywhere.  There are some related constants, comments, and methods in the ROS/Karel code, but they don't adhere strictly to these names either.